app/assets/javascripts/surveys/modules/Survey.js

```diff
@@ -525,7 +529,7 @@ const Survey = {
       const $conditionalQuestion = $(question);
       let $question = $($(question).parents(BLOCK_CONTAINER_SELECTOR));
       const conditionalOptions = Utils.parseConditionalString($conditionalQuestion.data('conditional-question'));
-      const { question_id, value } = conditionalOptions;
```

Comment:
These changes to camelcase are unrelated, and I think they may break under-tested functionality as well. Please undo all the question_id vs. questionId changes.